maintenance.md

```diff
@@ -188,9 +188,13 @@ In Regular update, do the following as part of the update of each CRD-providing
 ![Regular Update](./regular_update.svg)
 
 1. Check [releases](https://github.com/jetstack/cert-manager/releases) for changes.
-2. Update the `version` argument on the `build-cert-manager` job in the CircleCI `main` workflow.
-3. Update image tag in `README.md`.
-4. Update `BRANCH` and `TAG` files.
+2. Check whether manually applied patches have been included in the new release and remove them accordingly.
+   1. If patches are still needed, synchronize the forked repository (https://github.com/cybozu-go/cert-manager)
+   2. Create and checkout a new branch from the tag named vX.Y.Z
```

Comment:
Guessing from the last operation, we check out a new branch `vX.Y.Z-neco` from vX.Y.Z, cherry-pick the patch and push it. Lastly, create a new tag named vX.Y.Z-neco-longtimeout.1 and push it. Am I correct?

Reply:
Thank you.
Yes, you are correct.
I should fix it.